rules/integrations/okta/initial_access_okta_user_sessions_started_from_different_geolocations.toml

```diff
@@ -4,63 +4,104 @@ integration = ["okta"]
 maturity = "production"
 min_stack_comments = "Breaking change in Okta integration bumping version to ^2.0.0"
 min_stack_version = "8.10.0"
```

Comment:
https://github.com/elastic/detection-rules/pull/3797#discussion_r1647202783

Reply:
Referencing the min stack review from a cross PR as i observed this is also an ESQL change.

Reply:
Updated, thanks for pointing that out!